cmd/neofs-cli/modules/control/notary_request.go

```diff
@@ -0,0 +1,122 @@
+package control
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	rawclient "github.com/nspcc-dev/neofs-api-go/v2/rpc/client"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/commonflags"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/key"
+	ircontrol "github.com/nspcc-dev/neofs-node/pkg/services/control/ir"
+	ircontrolsrv "github.com/nspcc-dev/neofs-node/pkg/services/control/ir/server"
+	"github.com/spf13/cobra"
+)
+
+const notaryMethodFlag = "method"
+
+var notaryRequestCmd = &cobra.Command{
+	Use:   "request",
+	Short: "Create and send a notary request",
+	Long: "Create and send a notary request with one of the following methods:\n" +
+		"- newEpoch, transaction for creating of new NeoFS epoch event in FS chain, no args\n" +
+		"- setConfig, transaction to add/update global config value in the NeoFS network, 1 arg in the form key=val\n" +
+		"- removeNode, transaction to move nodes to the Offline state in the candidates list, 1 arg is the public keys of the node",
+	RunE: notaryRequest,
+}
+
+func initControlNotaryRequestCmd() {
+	initControlFlags(notaryRequestCmd)
+
+	flags := notaryRequestCmd.Flags()
+	flags.String(notaryMethodFlag, "", "Requested method")
+}
+
+func notaryRequest(cmd *cobra.Command, args []string) error {
+	ctx, cancel := commonflags.GetCommandContext(cmd)
+	defer cancel()
+
+	pk, err := key.Get(cmd)
+	if err != nil {
+		return err
+	}
+
+	cli, err := getClient(ctx)
+	if err != nil {
+		return err
+	}
+
+	req := new(ircontrol.NotaryRequestRequest)
+	body := new(ircontrol.NotaryRequestRequest_Body)
+	req.SetBody(body)
+
+	method, _ := cmd.Flags().GetString(notaryMethodFlag)
+	body.SetMethod(method)
+
+	switch method {
+	case "newEpoch":
+		if len(args) > 0 {
+			cmd.Println("method 'newEpoch', but the args provided, they will be ignored")
+		}
+	case "setConfig":
+		if len(args) != 1 {
+			return fmt.Errorf("invalid number of args provided for 'setConfig', expected 1, got %d", len(args))
+		}
+
+		bodyArgs := make([]string, 0, len(args)*2)
+		for _, arg := range args {
+			k, v, err := parseConfigPair(arg)
+			if err != nil {
+				return err
+			}
+
+			bodyArgs = append(bodyArgs, k, fmt.Sprint(v))
+		}
+		body.SetArgs(bodyArgs)
+	case "removeNode":
+		if len(args) != 1 {
+			return fmt.Errorf("invalid number of args provided for 'removeNode', expected 1, got %d", len(args))
+		}
+		body.SetArgs(args)
```

Comment:
Check (deserialize) the key and push it in binary, please.

Reply:
So, do I need to make `args` in the request in bytes for this?

Reply:
You mean protobuf-wise? Maybe, I'm not sure what's the protobuf difference between string and bytes. Anyway we don't want to pass hex values over this interface.

Reply:
the only diff is that `string` must be UTF-8 data https://protobuf.dev/programming-guides/encoding/#cheat-sheet-key. And `string` is encoded exactly like `bytes` https://protobuf.dev/programming-guides/encoding/#structure (https://github.com/nspcc-dev/neofs-sdk-go/blob/5f693d22cfe6350060e546e383c293317253352a/internal/proto/encoding.go#L163-L166)

Reply:
Kinda as expected. Then `bytes` that is.